### ISSUE TITLE
Anglo-centricism

### ISSUE THREAD

**Issue by shirakaba** [coryetzkorn/twitter-algorithm#3]
If I wanted to praise Elon in non-Roman scripts like Japanese (イーロン) or Chinese (伊隆) I wouldn't get any points. What gives?

**Comment by cooljeanius** [comment]
makes sense, given that Elon's rule of Twitter has been very Anglo-centric so far
